addons/onion-skinning/addon.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Onion Skinning",
+  "description": "Shows transparent overlays of previous costumes when editing a costume.",
+  "credits": [
+    {
+      "name": "GarboMuffin"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userscript.css",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "settings": [
+    {
+      "id": "default",
+      "name": "Enable by default",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "id": "layers",
+      "name": "Layers",
+      "type": "positive_integer",
+      "default": 1
+    },
+    {
+      "id": "opacity",
+      "name": "Opacity levels",
+      "type": "string",
+      "default": "0.25,0.15,0.05"
```

Comment:
You might want to convert this into three separate `positive_integer`s, using % instead of decimals. However at the moment there's no way to declare a maximum value (here it would be 100) so we might as well add another setting type for this usecase.
Another alternative might be being able to change the opacity from the UI itself, which might make a bit more sense since the opacity might depend on what you're drawing above of.